src/pedestal_toolbox/cors.clj

```diff
@@ -0,0 +1,7 @@
+(ns pedestal-toolbox.cors)
+
+(defn domain-matcher-fn [list-of-strings]
+  (let [patterns (map re-pattern list-of-strings)]
```

Comment:
Should we convert strings to regexes before they get here and have this fn expect regexes?